otri/utils/key_handler.py

```diff
@@ -213,4 +237,28 @@ def round_op(string, digits=digits):
             return rounded_number
         except ValueError:
             return string
-    return apply_deep_values(data, lambda x: round_op(x) if isinstance(x, float) else x)
\ No newline at end of file
+    return apply_deep_values(data, lambda x: round_op(x) if isinstance(x, float) else x)
+
+
+def round_shallow(data: Mapping, keys: List, digits: int = 3) -> Mapping:
+    '''
+    Rounds the passed keys' values of the dict or list to a certain digit.
+    The method only checks the first "layer" of keys, without going deeper.
+    All keys' values must be numbers.
+
+    Raises:
+        TypeError if one of the given key is not a float
+        KeyError if one of the give key is not in data
+    '''
+    if isinstance(data, List):
+        return [__round_shallow_dict(element, keys, digits) for element in data]
+    return __round_shallow_dict(data, keys, digits)
+
+
+def __round_shallow_dict(data: Mapping, keys: List, digits: int = 3) -> Mapping:
+    for key in keys:
+        try:
+            data[key] = round(float(data[key]), ndigits=digits)
+        except:
```

Comment:
```suggestion
        except error:
            log.e(error)
```
Regarding above Codacy issue

Reply:
Had no idea about it, nice thanks.